hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -277,9 +278,14 @@ public ContainerCommandResponseProto sendCommand(
     List<DatanodeDetails> datanodeList = pipeline.getNodes();
     HashMap<DatanodeDetails, CompletableFuture<ContainerCommandResponseProto>>
             futureHashMap = new HashMap<>();
+    ContainerCommandRequestProto.Builder builder = ContainerCommandRequestProto.newBuilder(request);
```

Comment:
nit: create a helper function something like`getVersionRequest()` and reuse it in other places.